frontend/test/metabase-db/mysql/query.cy.spec.js

```diff
@@ -0,0 +1,122 @@
+import {
+  signInAsAdmin,
+  restore,
+  modal,
+  signInAsNormalUser,
+} from "__support__/cypress";
+
+function addMySqlDatabase() {
+  cy.request("POST", "/api/database", {
+    engine: "mysql",
+    name: "MySQL",
+    details: {
+      host: "localhost",
+      dbname: "sample",
+      port: 3306,
+      user: "metabase",
+      password: "metasample123",
+      authdb: null,
+      "additional-options": null,
+      "use-srv": false,
+      "tunnel-enabled": false,
+    },
+    auto_run_queries: true,
+    is_full_sync: true,
+    schedules: {
+      cache_field_values: {
+        schedule_day: null,
+        schedule_frame: null,
+        schedule_hour: 0,
+        schedule_type: "daily",
+      },
+      metadata_sync: {
+        schedule_day: null,
+        schedule_frame: null,
+        schedule_hour: null,
+        schedule_type: "hourly",
+      },
+    },
+  });
+}
+
+describe("mysql > user > query", () => {
+  before(() => {
+    restore();
+    signInAsAdmin();
+    addMySqlDatabase();
+  });
+
+  beforeEach(() => {
+    signInAsNormalUser();
+  });
+
+  it("can query a My SQL database as a user", () => {
+    cy.visit("/question/new");
+    cy.contains("Simple question").click();
+    cy.contains("MySQL").click();
+    cy.contains("Orders").click();
+    cy.contains("Showing first 2,000 rows");
```

Comment:
```suggestion
```

I'd advise against using number of rows displayed in assertions. I've run into a couple of problems with some older tests where that assertion had to be removed (number of rows changes based on the screen size and is fragile as such).

Reply:
```suggestion
    cy.contains("Showing first");
```